### ISSUE TITLE
freebsd support

### ISSUE THREAD

**Issue by dcou** [ryanmjacobs/c#40]
I got your script working on freebsd with multiple parameters using this as my first line

#!/usr/bin/env -S c "-I/usr/local/include/ -g -L/usr/local/lib/mysql/ -lmysqlclient --"

might be worth mentioning in the readme

**Comment by dcou** [comment]
Yes even for simple scripts, on FreeBSD 11, it has to be

#!/usr/bin/env c

#include <stdio.h>

int main(void) {
    printf("hello world\n");
    return 0;
}

my other more complex example is required when using env with multiple arguments

**Comment by dcou** [comment]
Yes, even for simple scripts, on FreeBSD 11 at least, it has to be

```
#!/usr/bin/env c

#include <stdio.h>

int main(void) {
    printf("hello world\n");
    return 0;
}
```
my other more complex example is when you need to pass multiple arguments using env

**Comment by andy5995** [comment]
@ryanmjacobs You could add a CI FreeBSD test using https://github.com/vmactions/freebsd-vm
